src/components/AuthContainer.ts

```diff
@@ -15,11 +15,11 @@ import TermsOfService from 'src/pages/TermsOfService';
 const AuthContainer = ({ children }) => {
   const { name, public: isPublic } = useRoute();
   const { isSignedIn, registrationStatus, termsOfService, profile } = useStore(authStore);
-  const displayTosPage = isSignedIn && termsOfService.permitsSystemUsage === false;
+  const displayTosPage = isSignedIn === true && (termsOfService as any).permitsSystemUsage === false;
```

Comment:
#4258 should remove the need for this `any`, right?

Reply:
yep, I can make that change on whichever one merges 2nd.